File: AkitaModel/Portal/PatientTests.cs
using System.Collections.Generic;

namespace Skyware.Lis.AkitaModel.Portal;

/// <summary>
/// Represents a patient and distinct test entries.
/// </summary>
public class PatientTests
{

    /// <summary>
    /// Patient.
    /// </summary>
    public Patient Patient { get; set; }

    /// <summary>
    /// The test summaries the patient.
    /// </summary>
    public IList<TestSummary> TestSummaries { get; set; }

}

File: AkitaMotelTS/src/Portal/PatientTests.ts
/**
 * Skyware.Lis.AkitaModel.Portal
 */

import { Patient } from '../Patient';
import { TestSummary } from './TestSummary';

/**
 * Represents a patient and distinct test entries.
 */
export class PatientTests {
  /**
   * Patient.
   */
  patient?: Patient;

  /**
   * The test summaries the patient.
   */
  testSummaries?: TestSummary[];
}
